assets/src/blocks/ActionsList/index.js

```diff
@@ -62,22 +56,32 @@ export const registerActionsList = () => {
             },
           },
         }],
-        ['core/post-template', {}, [
+        ['core/post-template', {lock: {move: true, remove: true}}, [
           ['core/post-featured-image', {isLink: true}],
           ['core/group', {}, [
             ['core/post-terms', {term: 'post_tag', separator: ' '}],
             ['core/post-title', {isLink: true}],
             ['core/post-excerpt'],
           ]],
-          ['core/group', {layout: {type: 'flex', justifyContent: 'right'}}, [
+          ['core/group', {className: 'read-more-nav'}, [
             ['core/read-more', {
               className: 'btn btn-small btn-primary',
               content: __('Take Action', 'planet4-blocks-backend'),
             }],
           ]],
         ]],
+        ['core/buttons', {
+          className: 'carousel-controls',
+          lock: {move: true},
+          layout: {type: 'flex', justifyContent: 'space-between', orientation: 'horizontal', flexWrap: 'nowrap'},
+        },
+        [
+          ['core/button', {className: 'carousel-control-prev', text: __('Prev', 'planet4-blocks-backend')}],
+          ['core/button', {className: 'carousel-control-next', text: __('Next', 'planet4-blocks-backend')}],
+        ],
+        ],
+        ['core/query-pagination', {lock: {move: true}}],
```

Comment:
Hey @mleray, @comzeradd, Maybe I was not there during the original conversations, but I am curious as to why we implemented pagination on this block, seeing the discussion is to have a button that leads to a listing page or something. Is there some thing I am missing, just for clarity?

Reply:
The [initial ticket](https://jira.greenpeace.org/browse/PLANET-7251) specified that we shouldn't implement the "Load more" functionality, so I guess this was the easiest option? 🤔

Reply:
Oh yes you are right. We shouldn’t implement pagination here. We will just add a “Read All” button in a follow up ticket. It’s out of scope anyway for the carousel layout.

Reply:
Let’s remove it completely from this PR